app/bundles/CoreBundle/Config/config.php

```diff
@@ -1831,6 +1831,9 @@
         'transliterate_page_title'  => false,
         'cors_restrict_domains'     => true,
         'cors_valid_domains'        => [],
+        'headers_sts'               => false,
```

Comment:
I'm missing the `preload` directive there, which is part of the spec.
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security

Reply:
@mollux I added `preload` directive in the last commit [e5a6e8c](https://github.com/mautic/mautic/pull/11608/commits/e5a6e8c9d39d266476414748143e9a2fd70c510d). Please review the changes again.